src/main/java/org/elasticsearch/transport/netty/NettyTransport.java

```diff
@@ -110,76 +110,44 @@
     public static final String CONNECTIONS_PER_NODE_REG = "transport.connections_per_node.reg";
     public static final String CONNECTIONS_PER_NODE_STATE = "transport.connections_per_node.state";
     public static final String CONNECTIONS_PER_NODE_PING = "transport.connections_per_node.ping";
-    private static final String DEFAULT_PORT_RANGE = "9300-9400";
-
-    private final NetworkService networkService;
-    final Version version;
-
-    private final boolean blockingClient;
-    private final TimeValue connectTimeout;
-    private final ByteSizeValue maxCumulationBufferCapacity;
-    private final int maxCompositeBufferComponents;
-    final boolean compress;
-    private final ReceiveBufferSizePredictorFactory receiveBufferSizePredictorFactory;
-    private final int workerCount;
-    private final ByteSizeValue receivePredictorMin;
-    private final ByteSizeValue receivePredictorMax;
-
-    final int connectionsPerNodeRecovery;
-    final int connectionsPerNodeBulk;
-    final int connectionsPerNodeReg;
-    final int connectionsPerNodeState;
-    final int connectionsPerNodePing;
-
-    /*
-    final int workerCount;
-    final int bossCount;
-
-    final boolean blockingServer;
-
-    final String port;
-
-    final String bindHost;
-
-    final String publishHost;
-
-    final int publishPort;
-
-    final boolean compress;
-
-    final TimeValue connectTimeout;
-    final String tcpNoDelay;
-    final String tcpKeepAlive;
-    final Boolean reuseAddress;
-
-    final ByteSizeValue tcpSendBufferSize;
-    final ByteSizeValue tcpReceiveBufferSize;
-
-    final int connectionsPerNodeRecovery;
-    final int connectionsPerNodeBulk;
-    final int connectionsPerNodeReg;
-    final int connectionsPerNodeState;
-    final int connectionsPerNodePing;
-
-    final ByteSizeValue maxCumulationBufferCapacity;
-    final int maxCompositeBufferComponents;
-    */
-
-    final BigArrays bigArrays;
-    private final ThreadPool threadPool;
-    private volatile OpenChannelsHandler serverOpenChannels;
-    private volatile ClientBootstrap clientBootstrap;
+    protected static final String DEFAULT_PORT_RANGE = "9300-9400";
```

Comment:
I think it's fine to make this public and name it `DEFAULT_TRANSPORT_PORT_RANGE`?